pycbc/events/events.py

```diff
@@ -447,6 +447,8 @@ def __setitem__(self, name, data):
             f['bank_chisq_dof'] = self.events['bank_chisq_dof']
             f['cont_chisq'] = self.events['cont_chisq']
             f['end_time'] = self.events['time_index'] / float(self.opt.sample_rate) + self.opt.gps_start_time
+            #if self.opt.psdvar_short_segment is not None:
```

Comment:
Before merging, should probably remove the debug version.

Reply:
yep, removed